src/index.js

```diff
@@ -1,18 +1,19 @@
 import request from 'request-promise'
 import fs from 'fs'
 
-const BASE_SENTRY_URL = 'https://sentry.io/api/0/projects'
+const BASE_SENTRY_URL = 'https://sentry.io/api/0'
```

Comment:
Worth noting that this is technically a breaking API change, anyone that was providing the `baseSentryURL` config option would have to drop `/projects` from the end of it. I don't really see a good way around that since the URL now has to start with `/organizations`; it seems to me like `/projects` doesn't really belong in the base anyway (which is just pointing to the endpoint of their sentry installation).

Reply:
I decided to add a check to strip any `/projects` suffix from the URL, to prevent the need for a major version bump.